app/assets/stylesheets/bank_accounts.css.scss

```diff
@@ -17,6 +17,14 @@
   background-color: $positive-color !important;
 }
 
+.positive.darken {
```

Comment:
@yunixon don't call it `darken`. Use meaning name. Something like it was before `new-transaction`. If you want to use style for several classes, then write them all here `.new-transasaction, .total`. Or use mixin.